fixtures/sample-project-content/src/sample-collection/sample-component-fragment/fragment-configuration.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@victorg1991 can you add fixtures for projects with/without fragment-configuration?